ngrok/src/session.rs

```diff
@@ -1007,6 +1030,23 @@ impl Session {
     }
 }
 
+pub(crate) fn host_certs_tls_config() -> Result<rustls::ClientConfig, &'static io::Error> {
+    // The root certificate store, lazily loaded once.
+    static ROOT_STORE: Lazy<Result<RootCertStore, io::Error>> = Lazy::new(|| {
+        let der_certs = rustls_native_certs::load_native_certs()?
+            .into_iter()
+            .collect::<Vec<_>>();
+        let mut root_store = RootCertStore::empty();
+        root_store.add_parsable_certificates(der_certs);
+        Ok(root_store)
+    });
+
+    let root_store = ROOT_STORE.as_ref()?;
+    Ok(rustls::ClientConfig::builder()
+        .with_root_certificates(root_store.clone())
```

Comment:
nice!

Reply:
Moved all this over from the tunnel_ext to make it a utility, so not my code 😄